.github/workflows/tests.yaml

```diff
@@ -178,7 +179,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/darwin-x64-chip-tool-${BUILD_VARIANT}${CHIP_TOOL_VARIANT}/chip-tool \
-                     --target-skip-glob '{TestGroupMessaging,TV_*}' \
+                     --target-skip-glob '{TestGroupMessaging,TestGroupDemoCommand,TestGroupDemoConfig,TV_*}' \
```

Comment:
should the skip glob be `TestGroup*` ? Or are there tests included with that prefix that pass?

Reply:
No, other tests need to be run on darwin like `TestGroupsCluster` and `TestGroupKeyManagementCluster`.